extension/httpfs/s3fs.cpp

```diff
@@ -812,7 +830,7 @@ void S3FileHandle::Initialize(FileOpener *opener) {
 }
 
 bool S3FileSystem::CanHandleFile(const string &fpath) {
-	return fpath.rfind("s3://", 0) == 0;
+	return this->IsRemoteFile(fpath);
```

Comment:
You're right, it'll be clearer in that way. I decided to use the existing function to keep the code more compact since the http paths don't affect it, but I agree it's better to change it. Thanks for the comment.